src/main/java/seedu/address/logic/parser/task/TaskCliSyntax.java

```diff
@@ -9,7 +9,8 @@
 
     /* Prefix definitions */
     public static final Prefix PREFIX_DESCRIPTION = new Prefix("d/");
-    public static final Prefix PREFIX_ROOM_NUMBER = new Prefix("r/");
     public static final Prefix PREFIX_DUE_DATE = new Prefix("dd/");
+    public static final Prefix PREFIX_ROOM_NUMBER = new Prefix("r/");
```

Comment:
Is it better to use the prefix_room_number from RoomCliSyntax instead since room number is an attribute of room?

Reply:
Good point. Will use that instead 👍